src/main/java/com/cloudbees/jenkins/plugins/sshagent/mina/MinaRemoteAgentStarter.java

```diff
@@ -24,15 +24,23 @@
 
 package com.cloudbees.jenkins.plugins.sshagent.mina;
 
+import jenkins.security.MasterToSlaveCallable;
+
 import com.cloudbees.jenkins.plugins.sshagent.RemoteAgent;
+
 import hudson.model.TaskListener;
-import hudson.remoting.Callable;
 import hudson.remoting.Channel;
 
 /**
  * Callable to start the remote agent.
  */
-public class MinaRemoteAgentStarter implements Callable<RemoteAgent, Throwable> {
+public class MinaRemoteAgentStarter extends MasterToSlaveCallable<RemoteAgent, Throwable> {
+
+    /**
+     * 
+     */
+    private static final long serialVersionUID = 2909967505958103050L;
```

Comment:
Never increment/change unless you are making a backwards incompatible change. I am fairly certain this should be `-7956623493695287408L` but it needs checking. The changes to this class are not incompatible, so we should ensure that the version is the same as before the changes